### ISSUE TITLE
Reformat user guide with "real" elements

### ISSUE THREAD

**Issue by lauraykerr** [CDCgov/prime-simplereport#3997]
## Area or Component
User guide v 1.6 PDF

## Quick Summary
Much of the content in the user guide is not formatted with proper headers, lists, etc. 

## Accessibility Violation
PDFs should be formatted with real headers, lists, and tables at the source document, resulting in a properly tagged PDF that can be read by screen readers.

## Action Requested
Add proper headers and list elements to the source doc. 

## Source Code (optional)
[1.6 SimpleReport user guide](https://docs.google.com/document/d/1Hh3Gj4QOK2fVPNfrwd9nHSDO-waALcWHcHsGcHqudyM/edit?usp=sharing)

**Comment by lauraykerr** [comment]
## Area or Component
User guide v 1.6 PDF

## Quick Summary
Much of the content in the user guide is not formatted with proper headers, lists, etc. 

## Accessibility Violation
PDFs should be formatted with real headers, lists, and tables at the source document, resulting in a properly tagged PDF that can be read by screen readers.

## Action Requested
Add proper headers and list elements to the source doc. 

## Source Code (optional)
[1.6 SimpleReport user guide](https://docs.google.com/document/d/1Hh3Gj4QOK2fVPNfrwd9nHSDO-waALcWHcHsGcHqudyM/edit?usp=sharing)
